azureeyemodule/app/model/objectdetector.cpp

```diff
@@ -90,6 +90,58 @@ bool ObjectDetector::pull_data(cv::GStreamingCompiled &pipeline)
     return true;
 }
 
+bool ObjectDetector::pull_data_uvc_video(cv::GStreamingCompiled &pipeline)
+{
+    // All of our object detector models have the same graph outputs.
+    // These are the nodes from the raw camera frame path.
+    cv::optional<cv::Mat> out_bgr;
+
+    // These are th enodes from the neural network inference path.
```

Comment:
While you are fixing other typos, you might as well fix this one too: `These are th enodes` -> `These are the nodes`. Thanks

Reply:
good catch. thanks